scim-2.0.sgnl.yaml

```diff
@@ -395,15 +395,8 @@ relationships:
     name: Manager
     fromAttribute: Users.$..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].manager.value
     toAttribute: Users.id
-  
-  # The internal `parent` relationship cannot be directly referenced in `path` relationship.
-  # Create a dummy relationship to represent the `parent` relationship.
-  # The fromAttribute and toAttribute does not matter. Only entities should be specified correctly.
-  # We rely on the `parent` relationship already created during ingestion.
-  ParentOfGroupMember: 
-    name: parent
-    fromAttribute: groups.displayName
-    toAttribute: Users.id
+  ParentOfGroupMember:
+    childEntity: groups
```

Comment:
The format specified here assumes the FE will implicitly create the relationship with a name of `parent` (all lowercase). This field is not customizable so I figured this should not be exposed to the user.

This also assumes the `childEntity` is a valid child entity, otherwise this will fail on creation. The parent is automatically determined so it does not need to be specified here.

cc @ivanfong